generators/excel/main.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+import os
+import csv
+import openpyxl
+import sys
+from datetime import datetime
+
+OPENPYXL_MAJOR_VERSION = int(openpyxl.__version__[0])
+DATETIME_FORMAT = '%Y-%m-%dT%H:%M:%S%Z'
+# Minimum possible header.
+HEADER = (
+    'domain'
```

Comment:
A comma is missing after `'domain'`. The output needs to be regenerated as well since the `domain` and `entity` got merged in one string.